PowerValidatedSolutions.psm1

```diff
@@ -51387,7 +51388,7 @@ Function New-AriaNetworksNsxtDataSource {
         This example adds a NSX Manager as a new data source in VMware Aria Operations for Networks by using a NSX service account user with a password.
 
         .PARAMETER fqdn
-        The NSX Manager to add as a data source.
+        The fully qualified domain name of the NSX Manager to add as a data source.
```

Comment:
This change will require an update on the documentation .md file also

Reply:
Resolved this for Andy in 8789992.